pagopa/io-app
# chore: [IOBP-498] Adds stepper in payment's screens header

## Short description
This PR adds the stepper component in payment's screens header.

## List of changes proposed in this pull request
- Added `WalltPaymentHeader` component, to handle back navigation and display `Stepper` in the payment flow screens
- Added `react-navigation-view-pager` to handle navigation between screens in the payment flow
- Added `currentStep` to redux store with actions and selector to read and update it.
- Added `WalletPaymentMakeScreen` which displays the `ViewPager` with the payment flow screens.

## How to test
Within Wallet Payment Playgrounds, start a new payment flow and check that the stepper is correctly rendered

## Previews

| IOs | Android |
| --- | --- |
| <video src="https://github.com/pagopa/io-app/assets/6160324/71781d0f-801b-47fe-a03d-3a763285d926" /> | <video src="https://github.com/pagopa/io-app/assets/6160324/79af2822-c973-4d4c-a918-26226a033a13" /> |
